tests/queries/0_stateless/03241_orc_dictionary_encode.sql

```diff
@@ -0,0 +1,35 @@
+-- Tags: no-fasttest, no-parallel
+
+set input_format_orc_use_fast_decoder = 1;
+
+insert into function file(03241_data_without_dict.orc) 
```

Comment:
Let's use

```
currentDatabase() || '_03241_data_without_dict.orc'
```

as a file name. Otherwise, flaky check does not succeed. Then, we can remove the no-parallel.

Reply:
It doesn't look like it's working @alexey-milovidov  https://s3.amazonaws.com/clickhouse-test-reports/69481/af89ec5b82b59ca2fd754a9b081e8e9149448a25/stateless_tests_flaky_check__asan_.html